test/util/apply_obm_settings.py

```diff
@@ -0,0 +1,28 @@
+'''
+Copyright 2017, Dell EMC
+
+Author(s):
+George Paulos
+
+Script to apply OBM settings to all or selected nodes using BMC credentials in credentials_default.json
+
+Nodes or SKUs can be selected by using the '-sku' or '-nodeid' arguments at command line
+
+'''
+
+import sys
+import subprocess
+sys.path.append(subprocess.check_output("git rev-parse --show-toplevel", shell=True).rstrip("\n") + "/test/common")
+import fit_common
+from nose.plugins.attrib import attr
+
+
+@attr(all=True)
```

Comment:
Remember that the utility path is separate from the tests path. I think that the attr stuff can be eliminated from all of the utilities, its just not relevant. What do you think?